extensions/physics-nodes/CCPhysicsSpriteChipmunk2D.h

```diff
@@ -29,18 +29,14 @@
 #include "extensions/ExtensionExport.h"
 #include "base/CCEventListenerCustom.h"
 
-#if (CC_ENABLE_CHIPMUNK_INTEGRATION || CC_ENABLE_BOX2D_INTEGRATION)
-
 struct cpBody;
 class b2Body;
```

Comment:
Could be removed?

Reply:
Yes. Forgotten Thanks.

Reply:
Once you finish cleanup, I will merge

Reply:
This PR is the finish cleanup :)
You can merge plz.